jamf_protect_api/scripts/python/send_slack_webhook_analytics.py

```diff
@@ -0,0 +1,189 @@
+                },
+            },
+            {"type": "divider"},
+            {
+                "type": "section",
+                "fields": [
+                    {"type": "mrkdwn", "text": f"*Name:*\n{name}"},
+                    {"type": "mrkdwn", "text": f"*Severity:*\n{sev}"},
+                    {"type": "mrkdwn", "text": f"*Owner:*\n{analytic_source}"},
+                    {
+                        "type": "mrkdwn",
+                        "text": f"*Created:*\n{createdDate.strftime('%Y-%m-%d %H:%M')}",
+                    },
+                    {"type": "mrkdwn", "text": f"*Description:*\n{desc}"},
+                    {
+                        "type": "mrkdwn",
+                        "text": f"*Modified:*\n{updatedDate.strftime('%Y-%m-%d %H:%M')}",
+                    },
+                ],
+            },
+            {
+                "type": "section",
+                "text": {
+                    "type": "mrkdwn",
+                    "text": f"*<https://{PROTECT_INSTANCE}.protect.jamfcloud.com/analytics/{uuid}|View Analytic in Jamf Protect>*",
+                },
+            },
+        ]
+    }
+    slack_resp = requests.post(SLACK_URL + SLACK_TOKEN, json=payload)
+    print(slack_resp.status_code)
+
+
+def __main__():
+
+    # Get the access token
+    access_token = get_access_token(PROTECT_INSTANCE, CLIENT_ID, PASSWORD)
+
+    # Get all available Analytics
+    resp = make_api_call(PROTECT_INSTANCE, access_token, LIST_ANALYTICS_QUERY)
+
+    jamf_analytics_dict = {a["hash"]: a for a in resp["data"]["listAnalytics"]["items"]}
+
+    # Check is JSON file exists
+    if os.path.exists("/var/tmp/jamf-protect-analytics-update-slack-webhook.json"):
+
+        output = json.load(
+            open("/var/tmp/jamf-protect-analytics-update-slack-webhook.json", "r")
+        )
+
+        # Check diff between json file and new pull
+        new_or_updated_analytics = set(jamf_analytics_dict).difference(output)
+
+        if not new_or_updated_analytics:
+            print(f"No new or updated analytics.")
+            return
+
+        print(f"New or updated analytics available.")
+        for a_hash in new_or_updated_analytics:
+            new_analytic = jamf_analytics_dict[a_hash]
+            if any(
+                new_analytic["uuid"] == old_analytic["uuid"]
+                for old_analytic in output.values()
+            ):
+                new_analytic["analyticType"] = "Updated"
+            else:
+                new_analytic["analyticType"] = "New"
+            slack_webhook(
+                new_analytic["uuid"],
+                new_analytic["name"],
+                new_analytic["severity"],
+                new_analytic["description"],
+                new_analytic["jamf"],
+                new_analytic["created"],
+                new_analytic["updated"],
+                new_analytic["analyticType"],
+            )
+
+    print(f"Generating JSON file for next run.")
```

Comment:
f-string not needed if you're not doing any string formatting
```suggestion
    print("Generating JSON file for next run.")
```